src/test/java/org/kiwiproject/registry/eureka/server/EurekaRegistryServiceIntegrationTest.java

```diff
@@ -117,22 +123,25 @@ void shouldRegisterAndReturnRegisteredInstance() {
             var expectedAppId = serviceInstance.getServiceName().toUpperCase(Locale.getDefault())
                     + "-" + APP_TIMESTAMP_FORMATTER.format(now);
 
+            var eurekaRegistry = EUREKA.getEurekaServer().getRegistry();
```

Comment:
Same thing here...you should determine which are common actions that could be hidden behind a nicer API in the extension.

Reply:
Created kiwiproject/embedded-eureka#41 to address this